### PR TITLE
ARUHA-512: fixed consumers limiting bug

### PR DESCRIPTION
**Pull request by v-stepanov**
@adyach @antban @rcillo @lmontrieux please review

### CODE REVIEW

File: src/main/java/org/zalando/nakadi/repository/zookeeper/ZkChildrenCache.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        }
+    }
+
+    public static ZkChildrenCache createCache(final CuratorFramework client, final String key) {
+        try {
+            // in some rare case the cache start can fail because the node can be removed
+            // in specific moment by other thread/instance, then we need to retry
+            return executeWithRetry(
+                    () -> {
+                        final ZkChildrenCache newCache = new ZkChildrenCache(client, key);;
```
Comment (antban) [inline src/main/java/org/zalando/nakadi/repository/zookeeper/ZkChildrenCache.java]: Haha! I found one redundant symbol on this line!

**Comment by antban** [comment]
:+1:

File: src/main/java/org/zalando/nakadi/repository/zookeeper/ZkChildrenCache.java (hunk 1 above)
Comment (v-stepanov) [inline src/main/java/org/zalando/nakadi/repository/zookeeper/ZkChildrenCache.java]: fixed

**Comment by antban** [comment]
:+1:

**Comment by rcillo** [comment]
:shipit:

**Pull request merged**